imports/api/entries/entries.js

```diff
@@ -0,0 +1,63 @@
+import { Meteor } from 'meteor/meteor';
+import { Mongo } from 'meteor/mongo';
+const Entries = new Mongo.Collection('entries');
+import SimpleSchema from 'simpl-schema';
+
+let Schema = new SimpleSchema({
+  userId: String,
+  category: Object,
+  'category.id': String,
+  'category.title': String,
+  tz: Object,
+  'tz.id': String,
+  'tz.title': String,
+  'tz.offset': String,
+  'tz.daylightSaving': String,
+  lookingFor: String,
+  oneLineIntro: String,
+  verified: Boolean,
+  active: Boolean,
+  preferences: {
+    type: Array,
+    optional: true,
+  },
+  'preferences.$': {
```

Comment:
What is `preferences.$` meant for?

Reply:
The dollar sign? It indicates the single item in an array. https://github.com/aldeed/simple-schema-js#more-shorthand

But i'm not sure the use case of preferences field though.

Reply:
OHH I understand now, thanks!